doc/api/http.md

```diff
@@ -987,6 +987,17 @@ or
 request.setHeader('Cookie', ['type=ninja', 'language=javascript']);
 ```
 
+When the value is a string an exception will be thrown if it contains
+characters outside the `latin1` encoding.
+
+If you need to pass UTF-8 characters in the value please encode the value
+using the [RFC 8187][] standard.
+
+```js
+const filename = 'Rock 🎵.txt';
+request.setHeader('Content-Disposition', `inline; filename*=utf-8''${encodeURIComponent(filename)}`);
```

Comment:
Using `inline` with a filename seems a bit weird to me, but feel free to ignore that suggestion
```suggestion
request.setHeader('Content-Disposition', `attachment; filename*=utf-8''${encodeURIComponent(filename)}`);
```